### ISSUE TITLE
Autosuggestion for linked entities (especially relevant for toponyms) is English only

### ISSUE THREAD

**Issue by simon-clematide** [impresso/impresso-frontend#909]
When entering "Bodensee", nothing shows up in linked entity autosuggestion because only the English terms are searched through. Meaning, I would need to search for "Lake of Constance" to get a suggestions for the linked entity. 
A fix would be to include at least the French and German names from wikidata in the searched index.
The same problem exists in the entities search page.
![image](https://user-images.githubusercontent.com/3041168/84385639-394bef80-abf0-11ea-8c34-9a1faafa7c91.png)

**Comment by e-maud** [comment]
Good point. This would mean retrieving de labels for all entities, and having another suggester for them. Apparently one can query multiple suggesters at the same time: https://lucene.apache.org/solr/guide/6_6/suggester.html#Suggester-MultipleDictionaries.1

`http://localhost:8983/solr/techproducts/suggest?suggest=true& \
    suggest.dictionary=mySuggester&suggest.dictionary=altSuggester&wt=json&suggest.q=elec`

**Comment by mduering** [comment]
fixed?
